.github/workflows/tests.yml

```diff
@@ -49,9 +50,12 @@ jobs:
           submodules: true
           path: omnigibson-src
 
+      - name: Install CUDA and PyTorch
+        run: micromamba install cuda=11.8 -c conda-forge
```

Comment:
I thought you said this doesn't work?

Reply:
Yeah ignore this. I will figure out this with Cem tomorrow together.